libmscore/album.cpp

```diff
@@ -0,0 +1,248 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "album.h"
+#include "score.h"
+#include "xml.h"
+#include "musescoreCore.h"
+
+namespace Ms {
+
+//---------------------------------------------------------
+//---------------------------------------------------------
+//
+//   AlbumItem
+//
+//---------------------------------------------------------
+//---------------------------------------------------------
```

Comment:
I don't think we need a comment chunk so large...
```
//---------------------------------------------------------
//   AlbumItem
//---------------------------------------------------------
```
is good already.

Reply:
If it isn't too much of a problem, I would like to keep it like that until I the PR is ready.

Reply:
OK